src/sass/app/trade.scss

```diff
@@ -467,6 +467,7 @@
             display: flex;  /* fix for iOS */
             justify-content: center;
             align-items: center;
+            z-index: 9; /* fix for MSEdge */
```

Comment:
Not sure if we need the comments here, since it's going to be applied on all browsers either way.